static/versionDetails.json

```diff
@@ -297,6 +297,11 @@
       "description": "A list of addresses that can be accessed without using a proxy.",
       "type": "list"
     },
+		{
```

Comment:
indentation on this block is off, but that's not visible to the user as far as I know